### ISSUE TITLE
Update view when drain is adopted

### ISSUE THREAD

**Issue by ryayak1460** [Code4HR/adopt-a-drain#30]
As a User, when I adopt a drain the screen is not updated and I don't know that something has happened.

If I refresh the page I can see the change.

Update the popup and the list.

**Comment by ryayak1460** [comment]
Looking at the live instance of Norfolk's Adopt-a-Drain, I see a 500 error in production when adopting a drain, but not in development.

Since the given response (a 500 response with an error object) does not match what the things controller update method can render (a 500 response with a json.errors object), I'm not sure what causes this, other than some possible issue with the mailer.

Can we see the Heroku logs or Docker logs?

**Comment by BretFisher** [comment]
Does it happen in our new autobuild of develop branch? https://norfolkdrains-develop.herokuapp.com/

Let me invite you to Heroku app for develop and see if you can take a look there.

**Comment by ryayak1460** [comment]
I can't test on the develop branch at the moment because the Google Maps API key doesn't recognise the `norfolkdrains-develop.herokuapp.com` domain, so we may have to get that situated first.

**Comment by BretFisher** [comment]
Right, we need @kmcurry to update the api key to include that 2nd url (if that's allowed on maps api keys)
